packages/common/src/api/tan-query/useRemixes.ts

```diff
@@ -89,7 +89,12 @@ export const useRemixes = (
     }),
     initialPageParam: 0,
     getNextPageParam: (lastPage: RemixesQueryData, allPages) => {
-      if (lastPage?.tracks?.length < pageSize) return undefined
+      const isSecondPage = allPages.length === 1
```

Comment:
hmm this is a bit weird, let's chat tomorrow about this structure a bit so i can understand how the pages work and everything